app/routes/events/components/EventEditor/index.js

```diff
@@ -376,7 +349,23 @@ function EventEditor({
                   </Tooltip>
                 </div>
               )}
-
+            {['NORMAL', 'INFINITE'].includes(event.eventStatusType) && (
+              <Tooltip content="Frist for påmelding - antall timer før arrangementet">
+                <Field
+                  key="registrationDeadlineHours"
+                  label="Påmelding stenger (standard 2 timer)"
```

Comment:
Why does this say `standard 2 timer`? From what I understand the form defaults to two becuase of the initialValues, or? :thinking:

Reply:
I thought I would point out that the standard close time is 2 hours before. It can be removed.